app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -280,6 +282,23 @@ export const AppLayoutSidebarLaunched = ({
                 </DisabledItem>
               )}
               <SidebarActions {...{ experiment, refetch }} />
+              <p className="edit-divider position-relative small my-2">
+                <span className="position-relative bg-light pl-1 pr-2 text-muted">
+                  External Links
+                </span>
+              </p>
+              <LinkExternal
+                className="mx-1 my-2 nav-item d-block text-dark font-weight-semibold w-100"
+                href="https://experimenter.info"
+              >
+                <BookIcon className="mr-2" /> Experimenter Documentation
+              </LinkExternal>
+              <LinkExternal
+                className="mx-1 my-2 nav-item d-block text-dark font-weight-semibold w-100"
+                href="slack://channel?team=T027LFU12&id=CF94YGE03"
```

Comment:
Also these URLs would have a better home in the list of [`EXTERNAL_URLS` in `lib/constants.ts`](https://github.com/anthonytran7757/experimenter/blob/2bd5270d8be51b74e0eee46921f419413f7cf8aa/app/experimenter/nimbus-ui/src/lib/constants.ts#L33-L60). I think we've been trying to gather them all there, since they tend to change over time while being repeated in multiple locations.